src/common/filesystembase.h

```diff
@@ -107,8 +107,11 @@ namespace FileSystem {
      * Equivalent to QFile::remove(), except on Windows, where it will also
      * successfully remove read-only files.
      */
-    bool OCSYNC_EXPORT remove(const QString &fileName, QString *errorString = 0);
+    bool OCSYNC_EXPORT remove(const QString &fileName, QString *errorString = 0, bool moveToTrash = false);
```

Comment:
I don't think the moveToTrash feature belong in the FileSystem::remove function.
Since we anyway call it only once moveToTrash=true, we can as well call "moveToTrash" instead.